services/Order/RefundAmountCalculator.php

```diff
@@ -50,7 +49,7 @@ public function calculate($params)
         }
 
         foreach ($params['productList'] as $product) {
-            $amount += \Tools::ps_round($product['amount'], PayPal::getPrecision());
+            $amount += (float) $product['amount'];
```

Comment:
Hello Bogdan, i don't really understand why we remove the round with precision after changing it on paypal.php main class.

Why we don't use it once corrected with correct currency ?

Rounding is almost done after adding every product to the count (in another function) ?

Reply:
Redundant action. In fact, rounding is done here:
https://github.com/202ecommerce/paypal/blob/6.4.2/classes/API/Request/PaypalOrderPartialRefundRequest.php#L44